README.md

```diff
@@ -43,6 +43,12 @@ Running `nscli` in your terminal will present you with the following options:
 
 Navigate the list using the arrow keys, then press Enter to run the selected script.
 
+## Node.js Support
+
+This package is compatible with Node versions higher than 16.x 
```

Comment:
I just ran with node 15.14 without issues

Reply:
I found this doc about the [||= operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR_assignment), compatible with Node.js >=15.